src/components/ApplicationDetails/ApplicationDetails.tsx

```diff
@@ -157,7 +157,7 @@ const ApplicationDetails: React.FC<HacbsApplicationDetailsProps> = ({ applicatio
               showModal<{ submitClicked: boolean }>(
                 applicationDeleteModal(application),
               ).closed.then(({ submitClicked }) => {
-                if (submitClicked) navigate('/stonesoup');
+                if (submitClicked) navigate('/stonesoup/applications');
```

Comment:
If the Workspace switcher PR goes in first then we need to updated this to `/workspaces`